hw2/04-api.js

```diff
@@ -1,7 +1,51 @@
 /** Exercise 04 - API **/
 
-const url = 'https://restcountries.com/v3.1/all';
+async function getData(){
+    try {
+        const response = await fetch(url);
+        const json = await response.json();
+        
+        let countries = parseJSONResponse(json);
+
+        let ol = document.getElementById("results");
+        populateListElements(ol, countries);
+    } catch (error) {
+        alert(`Error occured: ${error.message}`);
+        console.log(error);
+    }
+}
 
-// Add your code here
+/* 
+    Loops through the JSON and formats each element as a string
+    storing the country name and population. 
```

Comment:
Produces expected output perfectly. Code is well commented and easy to read. I can see no suggestions at all for change here looks great!

Reply:
Thanks for your inputs it was very informative. I have addressed all your comments in my latest [commit](https://github.com/juhiaugustine/fullstack-homework/pull/3/commits/863e40b3b1b02ee6444008d3ea4b426a59b3954f). Please have a look. @CzerPDX

Reply:
Your changes look beautiful! Approved.